openstack/castellum/alerts/openstack/errors.alerts

```diff
@@ -45,7 +45,7 @@ groups:
         severity: warning
         support_group: containers
         tier: os
-        playbook: 'docs/support/playbook/castellum/missing_database_metrics'
+        playbook: 'https://operations.global.cloud.sap/docs/support/playbook/castellum/missing_database_metrics'
```

Comment:
Also, as a more general question: In the past, we have been discouraged from specifying internal domain names in this repo because it constitutes information leaking about internal systems, which is being perceived as a risk re: social engineering attacks. I personally don't find that this argument holds water, but I do want to flag that this was a soft policy in the past. Is this pull request aligned with Secure Engineering in this regard?

Reply:
@edda @uwe-mayer we discussed this concern this morning

from what I remember it is not a concern since this domain name is already known to the public